statistics-rest/src/main/java/org/uniprot/api/statistics/StatisticAttributeConfig.java

```diff
@@ -0,0 +1,31 @@
+package org.uniprot.api.statistics;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.stereotype.Component;
+import org.uniprot.api.common.repository.search.facet.FacetConfig;
+import org.uniprot.api.common.repository.search.facet.FacetProperty;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+@Component
+@Getter
+@Setter
+@PropertySource({"classpath:statistics.properties"})
+@ConfigurationProperties(prefix = "statistics")
+public class StatisticAttributeConfig extends FacetConfig {
+
```

Comment:
Please add test for this class. You can see similar test in Proteome

Reply:
I just added it for the sake of consistency.

However, IMO, it's a kind of a waste of time since we are literally resting the spring framework's code and not really our code. Even if you see the test in Proteome, you could see it. It is nothing more than adding some line coverage.
Also, we might need to update tests, whenever we change the file.

The only supportive point I can see is: we can test whether we have added the properties correctly in the property file.

Overall, I would say not worth it.